course-mw/sunbird-util/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraDACImpl.java

```diff
@@ -23,7 +22,59 @@
 import org.sunbird.common.responsecode.ResponseCode;
 
 public class CassandraDACImpl extends CassandraOperationImpl {
-    
+
+
+  @Override
+  public Response getBlobAsText(RequestContext requestContext, String keySpace, String table, Map<String, Object> filters, List<String> properties) {
+    Response response = new Response();
+    Session session = connectionManager.getSession(keySpace);
+
+    StringBuilder sb = new StringBuilder();
```

Comment:
@pritha-tarento - why are we using a StringBuilder to create the query? we should use cassandra query builder classes like Select..

Reply:
query builder classes like Select does not looks like support the blob data type. Hence used String builder